apps/prairielearn/src/schemas/schemas/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { z, type ZodType } from 'zod';
+
+export function uniqueArray(schema: ZodType) {
```

Comment:
Need to check if this is picked up in conversion to JSONSchema correctly. If not, I might try `z.Set`.

Reply:
It was not; changed in [f9a0641](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/f9a06415176aa73894ac9fd409253a1c8b8bb165). JSONSchema will pick this up from a glance at source.